### DISCUSSION TITLE
Virtual monitor can only have four

### DISCUSSION THREAD

**Discussion by lxlalalala** [rustdesk/rustdesk#8875] [General]
### Bug Description

Virtual display, clicking 1, 2, 3 times has no response. It is not until the fourth time that four virtual monitor suddenly appear.Then if you click to reduce virtual monitor once, all virtual monitor will disappear.

### How to Reproduce

I'm not sure how it was triggered because I reinstalled both apps and still can't restore them.

### Expected Behavior

Click once to add a virtual monitor, and only one virtual monitor will appear normally

### Operating system(s) on local side and remote side

windows 11 -> windows 11

### RustDesk Version(s) on local side and remote side

1.2.7 -> 1.2.7

### Screenshots

Uploading 屏幕录制 2024-07-29 142541-.mp4…
![Uploading 屏幕录制 2024-07-29 142541-.gif…]()

**Comment by fufesou** [comment]
Thanks for your feedback. The screenshot is broken.

I just tested Win -> Win. It works as expected. Do you click the  button very fast?
